docs/source/stability_policy.rst

```diff
@@ -0,0 +1,132 @@
+Stability Policy
+================
+
+The ``python-telegram-bot`` library (PTB) tries to provides a stable interface to its users.
+As both the user base and the complexity of the library has grown significantly since PTBs conception, we introduced below explicit stability Policy with version NEXT.VERSION.
+
+Large parts of the :mod:`telegram` package are the Python representations of the Telegram Bot API, the stability policy of which PTB can not influence.
```

Comment:
```suggestion
Large parts of the :mod:`telegram` package are the Python representations of the Telegram Bot API, which stability PTB can not influence.
```

Reply:
should be "whose"?

Reply:
its not a person

Reply:
> its not a person

I think you may be confusing that with "who's"